cmd/install.go

```diff
@@ -18,19 +18,20 @@ type installCmd struct {
 }
 
 type installOpts struct {
-	force bool
+	force    bool
+	provider string
 }
 
 func newInstallCmd() *installCmd {
 	var root = &installCmd{}
 	// nolint: dupl
 	var cmd = &cobra.Command{
-		Use:           "install",
+		Use:           "install <url>",
 		Aliases:       []string{"i"},
-		Short:         "Installs the specified project",
+		Short:         "Installs the specified project from a url",
 		SilenceUsage:  true,
 		SilenceErrors: true,
-		Args:          cobra.MaximumNArgs(2),
+		Args:          cobra.MinimumNArgs(1),
```

Comment:
I don't believe this change is strictly necessary, as the number of `Args` is not the same as the number of flags. However, if it is needed to make something else work, I don't see a problem with keeping it.

Reply:
Oh, this change is unrelated to this PR. I just realized that we didn't configure that correctly and I'm fixing that here since `bin install` always requires a minimum of a 1 argument to work :D